features/signal/prevent_close/README.md

```diff
@@ -0,0 +1,18 @@
+# Pending signals prevents workflows from closing
+
+In Temporal, any pending signal for a given workflow execution will prevent that
```

Comment:
Don't think we need "In Temporal" in front of all these

Reply:
+1